datanode/api/trading_data_v2.go

```diff
@@ -3194,13 +3196,46 @@ func (t *tradingDataServiceV2) GetActiveNetworkHistoryPeerAddresses(_ context.Co
 	if t.networkHistoryService == nil {
 		return nil, apiError(codes.Internal, ErrNetworkHistoryNotEnabled, fmt.Errorf("network history is not enabled"))
 	}
-	addresses := t.networkHistoryService.GetActivePeerAddresses()
 
 	return &v2.GetActiveNetworkHistoryPeerAddressesResponse{
-		IpAddresses: addresses,
+		IpAddresses: t.networkHistoryService.GetActivePeerIPAddresses(),
 	}, nil
 }
 
+func (t *tradingDataServiceV2) NetworkHistoryStatus(_ context.Context, _ *v2.NetworkHistoryStatusRequest) (*v2.NetworkHistoryStatusResponse, error) {
+	connectedPeerAddresses, err := t.networkHistoryService.GetConnectedPeerAddresses()
+	if err != nil {
+		return nil, t.formatE(fmt.Errorf("failed to get connected peer addresses: %w", err))
+	}
+
+	// A subset of the connected peer addresses are likely to be copied to form another nodes peer set, randomise the list
+	// to minimise the chance that the same sub set are copied each time.
+	rand.Shuffle(len(connectedPeerAddresses), func(i, j int) {
```

Comment:
Not sure I get this. Are people not just going to ping the endpoint and copy the entire list into there new datanode's config?

Reply:
true, that may happen, I was optimistically thinking forward to the day when we have lots of peers, and a subset would be selected, just tyring to do what we can to ensure we don't have a dependency  on a central set of nodes.  Perhaps unnecessary.